src/components/LandingPage/LandingPageCallToAction/index.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react'
 import { CallToAction } from 'components/CallToAction'
 
-export const LandingPageCallToAction = () => {
+export const LandingPageCallToAction = (): JSX.Element => {
     return (
         <div className="ctas flex flex-col items-center justify-center w-full max-w-xl sm:mx-auto sm:flex-row">
-            <CallToAction icon="rocket" href="https://app.posthog.com/signup?src=home-hero">
+            <CallToAction icon="rocket" type="primary" to="/sign-up?src=home-hero">
```

Comment:
Are we hardcoding everyone to the new page? Why aren't we using a flag to determine where this should route to?

Reply:
This is just because of the jankiness of mounting Kea within SSR. For some reason I cannot get this to work unless a page refresh / navigation occurs. We (nearly — as you pointed out with your Loom) instantly redirect away from the page.